rocky/reports/views/generate_report.py

```diff
@@ -100,6 +103,21 @@ class ExportSetupGenerateReportView(GenerateReportStepsMixin, BreadcrumbsGenerat
     breadcrumbs_step = 6
     current_step = 4
 
+    def post(self, request, *args, **kwargs):
+        selected_plugins = request.POST.getlist("plugin", [])
+
+        client = get_katalogus(self.organization.code)
+        for selected_plugin in selected_plugins:
+            try:
+                client.enable_boefje_by_id(selected_plugin)
+            except HTTPError:
+                messages.error(
+                    request,
+                    _("An error occurred while enabling {}. The plugin is not available.").format(selected_plugin),
+                )
+                return self.post(request, *args, **kwargs)
+        return super().post(request, *args, **kwargs)
+
```

Comment:
Can you move this post to the `ReportFinalSettingsView` in base.py? Then it will be implemented for aggregate and multireport

Reply:
Hmm no I cant move it to there because these plugins are enabled only in the export step, and not taken all the way to the final step like the other settings.

And the button already says: enable and continue